src/main/java/mate/academy/onlinebookstore/dto/CreateBookRequestDto.java

```diff
@@ -3,14 +3,15 @@
 import jakarta.validation.constraints.Min;
 import jakarta.validation.constraints.NotNull;
 import java.util.Set;
-import org.hibernate.validator.constraints.ISBN;
+import lombok.experimental.Accessors;
 
+@Accessors(chain = true)
 public record CreateBookRequestDto(
         @NotNull
         String title,
         @NotNull
         String author,
-        @ISBN
+        @NotNull
```

Comment:
```suggestion
        @NotBlank
```

For all String types